src/services/requiredStandardsDatabaseService.ts

```diff
@@ -25,7 +25,7 @@ export class IvaDatabaseService {
    * @param euVehicleCategory the EU Vehicle Category, e.g M1, N1, MSVA
```

Comment:
Probably goes for any methods that have been refactored, but can you change the reference in the doc-blocks to Required Standards as well please.